db/install.xml

```diff
@@ -27,6 +27,8 @@
         <FIELD NAME="origin" TYPE="char" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="cli, cron, ws, etc."/>
         <FIELD NAME="ip" TYPE="char" LENGTH="45" NOTNULL="false" SEQUENCE="false" COMMENT="IP address"/>
         <FIELD NAME="realuserid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="real user id when logged-in-as"/>
+        <FIELD NAME="logstoreid" TYPE="int" LENGTH="10" NOTNULL="false" SEQUENCE="false" COMMENT="logstore_standard_log_id"/>
```

Comment:
I think the name should be `logstorestandardlogid` to avoid any ambiguity.

The `COMMENT` field contains underscores

Reply:
I have removed the underscores in the comment and changed this field to logstorestandardlogid and I also use that fieldname in upgrade.php